src/i18n/strings/en_EN.json

```diff
@@ -3770,6 +3770,7 @@
         "error_revoke_3pid_invite_title": "Failed to revoke invite",
         "hide_sessions": "Hide sessions",
         "hide_verified_sessions": "Hide verified sessions",
+        "ignore_button": "Block",
         "ignore_confirm_description": "All messages and invites from this user will be hidden. Are you sure you want to ignore them?",
         "ignore_confirm_title": "Ignore %(user)s",
```

Comment:
If 'block' is the new language we want to use, that should be applied to these other strings as well I assume